pallets/subtensor/src/lib.rs

```diff
@@ -1105,6 +1108,19 @@ pub mod pallet {
 			Self::do_registration(origin, netuid, block_number, nonce, work, hotkey, coldkey)
 		}
 
+		/// Associates the hotkey to this coldkey.
+		/// TODO(const): this call should be costly.
+		#[pallet::call_index(6)]
+		#[pallet::weight((Weight::from_ref_time(2_000_000_000) /// making this an expensive call?
```

Comment:
Triple-slash comments (`///`) are for doc-comments, so either double-slash or (better yet) inline comments (`/* ... */`) should be used instead, eg.

```diff
-#[pallet::weight((Weight::from_ref_time(2_000_000_000) /// making this an expensive call?
+#[pallet::weight((Weight::from_ref_time(2_000_000_000) /* making this an expensive call? */
```